tests/DependencyInjection/Compiler/AddProvidersPassTest.php

```diff
@@ -6,9 +6,11 @@
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\DependencyInjection\Argument\IteratorArgument;
 use Symfony\Component\DependencyInjection\Reference;
+use function class_exists;
 
 class AddProvidersPassTest extends TestCase
 {
+
```

Comment:
please revert the formatting changes to this file that are not related to the feature you add (this one is wrong. the newline things would not be necessary if createMock with the ::class would be used, but such cleanup should not be mixed into a new feature)